### ISSUE TITLE
loss increasing

### ISSUE THREAD

**Issue by mama110** [AlexeyAB/darknet#2098]
I'm training yolo with my own dataset on 2080 ti,  the average loss is increasing:
cfg file:
batch=256
subdivisions=64
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1
learning_rate=0.0001
burn_in=1000
max_batches = 1001
policy=steps
steps=40000,45000
scales=.1,.1

output:
Region 94 Avg IOU: 0.331726, Class: 0.412097, Obj: 0.692990, No Obj: 0.511341, .5R: 0.333333, .75R: 0.000000,  count: 3
Region 106 Avg IOU: 0.225691, Class: 0.561781, Obj: 0.448759, No Obj: 0.520575, .5R: 0.000000, .75R: 0.000000,  count: 3
Region 82 Avg IOU: -nan(ind), Class: -nan(ind), Obj: -nan(ind), No Obj: 0.524829, .5R: -nan(ind), .75R: -nan(ind),  count: 0
Region 94 Avg IOU: 0.268031, Class: 0.433397, Obj: 0.434378, No Obj: 0.511296, .5R: 0.000000, .75R: 0.000000,  count: 3
Region 106 Avg IOU: 0.323565, Class: 0.519937, Obj: 0.478082, No Obj: 0.521727, .5R: 0.000000, .75R: 0.000000,  count: 4
Region 82 Avg IOU: 0.158736, Class: 0.534290, Obj: 0.602303, No Obj: 0.525231, .5R: 0.000000, .75R: 0.000000,  count: 1
Region 94 Avg IOU: -nan(ind), Class: -nan(ind), Obj: -nan(ind), No Obj: 0.511679, .5R: -nan(ind), .75R: -nan(ind),  count: 0
Region 106 Avg IOU: 0.244585, Class: 0.501885, Obj: 0.522578, No Obj: 0.520937, .5R: 0.000000, .75R: 0.000000,  count: 6
Region 82 Avg IOU: -nan(ind), Class: -nan(ind), Obj: -nan(ind), No Obj: 0.525723, .5R: -nan(ind), .75R: -nan(ind),  count: 0
Region 94 Avg IOU: 0.391436, Class: 0.396084, Obj: 0.581649, No Obj: 0.510878, .5R: 0.333333, .75R: 0.000000,  count: 3
Region 106 Avg IOU: 0.269855, Class: 0.400612, Obj: 0.586980, No Obj: 0.521819, .5R: 0.000000, .75R: 0.000000,  count: 4
Region 82 Avg IOU: 0.383695, Class: 0.287710, Obj: 0.448198, No Obj: 0.525040, .5R: 0.500000, .75R: 0.000000,  count: 2
Region 94 Avg IOU: 0.482365, Class: 0.547003, Obj: 0.446980, No Obj: 0.510973, .5R: 0.000000, .75R: 0.000000,  count: 1
Region 106 Avg IOU: 0.322463, Class: 0.446175, Obj: 0.512809, No Obj: 0.521625, .5R: 0.142857, .75R: 0.000000,  count: 7
Region 82 Avg IOU: 0.393909, Class: 0.417008, Obj: 0.423460, No Obj: 0.525204, .5R: 0.333333, .75R: 0.000000,  count: 6

**Comment by mama110** [comment]
loss was decreasing when it ran 100 epoch.
